dockers/docker-bgp/minigraph_facts.py

```diff
@@ -0,0 +1,432 @@
+#!/usr/bin/env python
```

Comment:
minigraph_facts.py could be a python library that could be used by other config generators